pkg/apis/flow/v1alpha1/xmltojsontransformation_lifecycle.go

```diff
@@ -38,32 +49,32 @@ func (t *XMLToJSONTransformation) GetGroupVersionKind() schema.GroupVersionKind
 
 // GetConditionSet retrieves the condition set for this resource. Implements the KRShaped interface.
 func (t *XMLToJSONTransformation) GetConditionSet() apis.ConditionSet {
-	return xmlToJSONCondSet
+	return XMLToJSONCondSet
 }
 
 // InitializeConditions sets the initial values to the conditions.
 func (ts *XMLToJSONTransformationStatus) InitializeConditions() {
-	xmlToJSONCondSet.Manage(ts).InitializeConditions()
+	XMLToJSONCondSet.Manage(ts).InitializeConditions()
 }
 
 // MarkServiceUnavailable marks XMLToJSONTransformation as not ready with ServiceUnavailable reason.
 func (ts *XMLToJSONTransformationStatus) MarkServiceUnavailable(name string) {
-	xmlToJSONCondSet.Manage(ts).MarkFalse(
+	XMLToJSONCondSet.Manage(ts).MarkFalse(
 		apis.ConditionReady,
 		"ServiceUnavailable",
 		"Service %q is not ready.", name)
 }
 
 // MarkServiceAvailable sets XMLToJSONTransformation condition to ready.
 func (ts *XMLToJSONTransformationStatus) MarkServiceAvailable() {
```

Comment:
The Ready condition should never be set by us. Knative will set it automatically based on the other conditions in the ConditionSet.